pyproject.toml

```diff
@@ -45,4 +45,4 @@ testpaths = [
 line-length = 100
 
 # Enable Pyflakes `E` and `F` codes by default.
-select = ["E", "F", "Q"]
+select = ["E", "F", "I", "Q", "FA102"]
```

Comment:
:+1:

Reply:
I'm always reluctant to tweak other people's lint/formatting configs, but this seemed relatively innocuous/helpful 😅